compiler/luci/pass/src/ForwardTransposeOpPass.cpp

```diff
@@ -29,26 +29,52 @@ using namespace luci;
 namespace
 {
 
+// Return true if below conditions are met
+// 1. t->perm() is CircleConst
+// 2. t->perm() is S32
+bool check_perm(const CircleTranspose *t)
```

Comment:
This function is moved before `create_cloned_transpose`, because `create_cloned_transpose` uses this function in `assert`.